native_client/java/libdeepspeech/src/androidTest/java/org/mozilla/deepspeech/libdeepspeech/test/BasicTest.java

```diff
@@ -149,4 +151,20 @@ public void loadDeepSpeech_sttWithMetadata_withLM() {
         assertEquals("she had your dark suit in greasy wash water all year", decoded);
         m.freeModel();
     }
+
+    @Test
+    public void loadDeepSpeech_HotWord() {
+        DeepSpeechModel m = new DeepSpeechModel(modelFile);
+
+        for(int i = 0; i < word.size(); i++ ){
+
```

Comment:
nit: extra line non required

Reply:
nit: also missing space between `)` and `{` and remove space between `i++` and `)`

Reply:
inside the loop or above the loop?

Reply:
inside